tests/test_models.py

```diff
@@ -347,3 +349,195 @@ def test_clear_expired_tokens_with_tokens(self):
         clear_expired()
         expiredt = AccessToken.objects.filter(expires__lte=timezone.now()).count()
         assert expiredt == 0
+    
+    def test_clear_expired_tokens_with_tokens_before_invalid_format(self):
+        self.client.login(username="test_user", password="123456")
+        oauth2_settings.REFRESH_TOKEN_EXPIRE_SECONDS = 0
+        app = Application.objects.create(
+            name="test_app",
+            redirect_uris="http://localhost http://example.com http://example.org",
+            user=self.user,
+            client_type=Application.CLIENT_CONFIDENTIAL,
+            authorization_grant_type=Application.GRANT_AUTHORIZATION_CODE,
+        )
+        AccessToken.objects.create(
+            token="777",
+            expires=timezone.now()-timedelta(days=30),
+            scope=2,
+            application=app,
+            user=self.user,
+            created=timezone.now()-timedelta(days=40),
+            updated=timezone.now()-timedelta(days=40),
+            )
+        AccessToken.objects.create(
+            token="888",
+            expires=timezone.now()-timedelta(days=15),
+            scope=2,
+            application=app,
+            user=self.user,
+            created=timezone.now()-timedelta(days=20),
+            updated=timezone.now()-timedelta(days=20),
+            )
+        ttokens = AccessToken.objects.count()
```

Comment:
This name is not descriptive
```suggestion
        access_token_count = AccessToken.objects.count()
```
would be better.

The same applies to `expiredt` and `rtokens` in other tests.

I know this is used in other tests, but it not good for reading the code.